packages/allure-cucumberjs/src/BufferEncodingUtility.ts

```diff
@@ -0,0 +1,8 @@
+export const getBufferEncoding = (encoding: any): BufferEncoding => {
+  switch(encoding) {
+    case "BASE64":
+      return "base64";
+    default:
+    return "utf-8";
```

Comment:
I'm not entirely sure what the base buffer encoding should be if its not set.  CucumberJs only had two encoding options, "BASE64" and "IDENTITY".  The default node encoding is "utf-8" I believe, so I set the default here if it wasn't previously set.

Reply:
I removed this function as Buffer.isEncoding() does the check for a valid type automatically.